draft-ietf-jsonpath-base.md

```diff
@@ -1311,6 +1357,27 @@ JSONPath:
   addressed by the previous path fragment. Array indices always start
   at 0.
 
+# JSON Pointer
+
+This appendix is informative.
+
+JSONPath is not intended as a replacement for, but as a more powerful
+companion to, JSON Pointer {{RFC6901}}. The purposes of the two standards
+are different.
+
+JSON Pointer is for identifying a single value within a JSON document whose
+structure is known.
+
+JSONPath can identify a single value within a JSON document, for example by
+using a Normalized Path. But JSONPath is also a query syntax that can be used
+to search for and extract multiple values from JSON documents whose structure
+is known only in a general way.
+
+A Normalized JSONPath can be converted into a JSON Pointer by converting the syntax,
+without knowledge of any JSON document. The inverse is not generally true: a numeric
+path component in a JSON Pointer may identify a member of a JSON object or may index an array.
+For this conversion, knowledge of the structure of the JSON document is needed.
```

Comment:
```suggestion
For conversion to a JSONPath query, knowledge of the structure of the JSON document is needed to distinguish these cases.
```

Reply:
I'll include this in the next commit, thanks.